external/runtime/runtime.depproj

```diff
@@ -48,22 +48,6 @@
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="@(RefAndRuntimePackageReference)" />
```

Comment:
Adding a custom target for that sounds uncomfortable to me. I gave just adding a new depproj a try in https://github.com/dotnet/wcf/compare/master...dagood:corefx-ref-depproj and it seems to work. Does it make sense to factor it that way?